src/pcapi/routes/external/unsubscribe_users.py

```diff
@@ -0,0 +1,47 @@
+import ipaddress
+
+from flask import request
+from sqlalchemy.orm.exc import NoResultFound
+
+from pcapi import settings
+from pcapi.core.users.models import User
+from pcapi.flask_app import public_api
+from pcapi.models import db
+from pcapi.models.api_errors import ApiErrors
+
+
+whitelist_ip_range = ipaddress.IPv4Network("185.107.232.0/24")
```

Comment:
Je suppose que c'est le range d'IP de SendinBlue. Si oui :

```suggestion
SENDINBLUE_IP_RANGE = ipaddress.IPv4Network("185.107.232.0/24")
```

plus clair :)